### PR TITLE
added links

### PR DESCRIPTION
**Pull request by srowhani**
#PATCH#

### CODE REVIEW

File: addon/styles/addon.scss [hunk 1]
```diff
@@ -156,6 +156,7 @@
     display: flex;
     flex-direction: row;
     align-items: center;
+    text-align: initial;
```
Comment (rox163) [inline addon/styles/addon.scss]: whats this for?

**Comment by rox163** [comment]
:+1:

File: addon/styles/addon.scss (hunk 1 above)
Comment (srowhani) [inline addon/styles/addon.scss]: I wanted to use frost-link css without it center aligning text

**Comment by sglanzer** [comment]
Thanks!

**Pull request merged**